packages/react-composites/src/composites/CallComposite/components/buttons/Custom.tsx

```diff
@@ -79,7 +82,7 @@ export const generateCustomDrawerButtons = (
         {allButtonProps
           .filter((buttonProps) => buttonProps.placement === key)
           .map((buttonProps, i) => ({
```

Comment:
can id be added here also for consistency?

Reply:
To add id to the drawer menu items I needed to make a small addition to the _DrawerMenuItemProps.